app/controllers/queue_items_controller.rb

```diff
@@ -1,8 +1,9 @@
 class QueueItemsController < ApplicationController
   before_action :require_login
+  before_action :set_items, only: [:index, :update_queue]
```

Comment:
You set items in `:update_queue` but you are not using it :D

`current_user.queue_items` you access them manually anyway! Maybe reconsider deleting this and the method if you don't use it.